tests/test_namespace_classes.py

```diff
@@ -208,6 +208,41 @@ async def test_mapping_methods(self) -> None:
             obj.update(dict(new_value=False), **dict(new_value=True))
             assert obj["new_value"] == True
 
+    async def test_strange_slots_dict_interplay_exists_but_is_avoided(
+        self
+    ) -> None:
+        if not hasattr(self._type, "__slots__"):
+            return
+
+        try:
+            class MisusedSubclass(self._type):
+                __slots__ = ("attr_0", "attr_1", "__dict__")
+                slots_types = dict(attr_0=int, attr_1=int)
+        except TypeError:
+            class MisusedSubclass(self._type):
+                __slots__ = ("attr_0", "attr_1")
+                slots_types = dict(attr_0=int, attr_1=int)
+
+        bad_obj = MisusedSubclass(attr_0=0, attr_1=1)
```

Comment:
This object is the not-misused one. Why is it bad?

Reply:
This is clearer

https://github.com/rmlibre/aiootp/blob/fd652586117e90c5f87fd8c4e2a28a55ede17425/tests/test_namespace_classes.py#L226-L230

https://github.com/rmlibre/aiootp/blob/fd652586117e90c5f87fd8c4e2a28a55ede17425/tests/test_namespace_classes.py#L233-L234

Resolved by (https://github.com/rmlibre/aiootp/pull/11/commits/3b015fd56c10f66379524ff31824a8f7ff0ad1e0)